### DISCUSSION TITLE
Issue When Upgrading From 3.4.5 to 3.6.9

### DISCUSSION THREAD

**Discussion by mr1716** [netbox-community/netbox#15466] [Help Wanted!]
As the title says, I am upgrading from 3.4.5 to 3.6.9, as everything works great in 3.4.5. But for 3.6.9, I am getting a weird issue. The upgrade works as I can load the UI and load in data, but the issues arise with a custom plugin. 

Loading the plugins and netbox, I get the following error, "[54] ERROR:  relation "django_content_type" does not exist at character 106
netbox-postgresql   .... SELECT "django_content_type"."id", "django_content_type"."app_label", "django_content_type"."model" FROM "django_content_type" WHERE ( ...)"

And when I load in data to a custom built plugin, I get an error saying RequestError "has no field named 'child_value'", 'exception': 'FieldDoesNotExist'." The thing is that plugin and no other apps have that child_value in their models, and the code will work just fine when I use v3.4.5. 

I am sorta stuck after wiping the database clean and trying again, with the same result.

**Comment by candlerb** [comment]
Maybe the plugin isn't compatible with Netbox 3.6.  Disable the plugin in the `PLUGINS` variable in configuration.py to check.

The Netbox APIs are not stable: they change from version to version, so plugins and other integrations have to be changed to match. Look for "breaking changes" in the release notes for [v3.5.0](https://github.com/netbox-community/netbox/releases/v3.5.0) and [v3.6.0](https://github.com/netbox-community/netbox/releases/v3.6.0)

Contact the plugin author for help. If you're the plugin author, then spin up a fresh 3.6.9 and debug your plugin there.

Having said all that: "django_content_type" is still a real table, and it might be a postgres permissions issue (ownership of tables or schema). You might need `ALTER DATABASE netbox OWNER TO netbox;` and/or `GRANT CREATE ON SCHEMA public TO netbox;` - search for those terms in the discussion archive for more info.

**Reply by mr1716** [reply]
@candlerb, thanks. That did solve the django_content_type error. But not the plugin error, as I'll have to figure out what that would be with the developer.

**Reply by mr1716** [reply]
It's just weird because in Postgres, the child_value isnt even in the table.
